From 2a07c83cfcc8f6a513a6c0c138e6aec1cebab71f Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Wed, 12 May 2021 21:10:54 +0200
Subject: [PATCH] build-sys: meson: Add missing include dir

Otherwise building module-echo-cancel fails here:

    FAILED: src/modules/module-echo-cancel.so.p/echo-cancel_adrian-aec.c.o
    cc -Isrc/modules/module-echo-cancel.so.p -I. -I../pulseaudio -Isrc -I../pulseaudio/src -I/usr/include/orc-0.4 -flto=auto -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu11 -march=x86-64 -mtune=generic -O2 -p>
    ../pulseaudio/src/modules/echo-cancel/adrian-aec.c:30:10: fatal error: adrian-aec-orc-gen.h: No such file or directory
       30 | #include "adrian-aec-orc-gen.h"
          |          ^~~~~~~~~~~~~~~~~~~~~~

Part-of: <https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/552>
---
 src/modules/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/meson.build b/src/modules/meson.build
index f9b46f844..9c498a4e1 100644
--- a/src/modules/meson.build
+++ b/src/modules/meson.build
@@ -304,7 +304,7 @@ foreach m : all_modules
   mod = shared_module(name,
     sources,
     headers,
-    include_directories : [configinc, topinc],
+    include_directories : [configinc, topinc, include_directories('.')],
     c_args : [pa_c_args, server_c_args, '-DPA_MODULE_NAME=' + name.underscorify()] + extra_flags,
     install : true,
     install_rpath : rpath_dirs,
-- 
GitLab

